tests/Doctrine/Tests/ORM/Query/QueryTest.php

```diff
@@ -405,7 +401,7 @@ public function testResultCacheProfileCanBeRemovedViaSetter() : void
         $query->useResultCache(true);
         $query->setResultCacheProfile();
 
-        self::assertAttributeSame(null, 'queryCacheProfile', $query);
+        self::assertAttributeEmpty('queryCacheProfile', $query);
```

Comment:
This assert uses `empty()` and changes behavior of a test compared to the previous used check of `null`. But this property contains usually an object. Should we keep it that way?